portfolio/src/test/java/com/google/sps/DataServletTest.java

```diff
@@ -56,127 +64,232 @@ public void tearDown() {
     helper.tearDown();
   }
 
-  @Test 
-  public void testPOSTSingleComment() throws Exception {
-    DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
-    HttpServletRequest postRequest = mock(HttpServletRequest.class);
-    HttpServletResponse postResponse = mock(HttpServletResponse.class);
+  private String[] convertStringToArray(String responseOutput) {
```

Comment:
Per effective java item 28 - "prefer lists to arrays" unless there's a strong performance reason.

Reply:
done!